server/lib/sanitize/sanitize.js

```diff
@@ -34,7 +35,12 @@ const sanitize = (req, res, configs) => {
     }
   }
 
-  if (req.user) input.reqUserId = req.user._id
+  if (req.user) {
+    input.reqUserId = req.user._id
+    const reqUserAccessLevel = getUserAccessLevels(req.user)
+    input.reqUserHasAdminAccess = reqUserAccessLevel.includes('admin')
+    input.reqUserHasDataadminAccess = reqUserAccessLevel.includes('dataadmin')
```

Comment:
yagni ? i mean get your user access yourself controller, if you need any. Maybe move those to `user_access_levels` after `hasAdminAccess`

Reply:
done in 610e06b67